events/2024-09-16-bioinfo_bootcamp-2024.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: event
+title: 2024 Bioinformatics Bootcamp - The Open University
+description: |
```

Comment:
maybe some of this goes after the `---` at the end, and just keep a shorter description like "The School of Life, Health, and Chemical Sciences (LHCS) at The Open University (OU) is running a free, week-long Bioinformatics Bootcamp"